app/views/projects.html

```diff
@@ -8,52 +8,201 @@
+                  </a>
+                </div>
 
-          <div ng-if="!showGetStarted" class="container surface-shaded"> <!-- Safari needs surface-shaded at mobile -->
-            <div ng-if="(projects | hashSize) === 0" class="text-muted" style="margin-top: 50px;">Loading...</div>
-            <div ng-if="(projects | hashSize) !== 0" class="gutter-top">
-              <h1 style="display: inline-block;">Projects</h1>
-              <a ng-if="canCreate" href="create-project" style="margin-top: 10px;" class="btn btn-lg btn-primary pull-right">New Project</a>
-              <alerts alerts="alerts"></alerts>
-              <div ng-repeat="project in projects | orderByDisplayName track by project.metadata.name"
-                   class="animate-repeat">
-                <div row flex cross-axis="center" class="tile tile-project tile-click tile-flex">
-                  <div flex class="project-summary">
-                    <h2 class="project truncate">
-                      <a class="tile-target" href="project/{{project.metadata.name}}">{{(project | uniqueDisplayName : projects)}}</a>
-                      <span ng-if="project.status.phase != 'Active'" data-toggle="tooltip" title="This project has been marked for deletion." class="pficon pficon-warning-triangle-o" style="cursor: help; vertical-align: top; margin-left: 5px;"></span>
-                    </h2>
+                <div row cross-axis="center" class="project-bar pad-lg table-toolbar form-inline">
+                  <input class="projectSearch" placeholder="Search projects" ng-model="filterStr">
+
+                  <span class="mar-left-sm mar-right-sm">
+                    <span class="vertical-divider"></span>
+                  </span>
+
+                  <div class="project-sort">
+                    <div pf-sort config="sortConfig" class="sort-controls"></div>
+                  </div>
+
+                  <span class="hidden-xs mar-left-sm mar-right-sm">
+                    <span class="vertical-divider"></span>
+                  </span>
+
+                  <span flex class="hidden-xs"></span>
+
+                  <div>
+                    <a ng-if="canCreate" href="create-project" class="hidden-xs btn btn-md btn-primary pull-right">
+                      <span>New project</span>
+                    </a>
+                  </div>
+                  <alerts alerts="alerts"></alerts>
+                </div>
+              </div>
+
+              <div ng-repeat="project in projects | findProject:filterStr">
+                <div row cross-axis="center"
+                     class="tile tile-project tile-click tile-flex tile-project-border mar-left-xl mar-right-xl mar-bottom-none">
+                  <!-- TODO (MarkDeMaria): Insert Star funcionality here
+                  <div>
+                    <i class="fa fa-star-o" aria-hidden="true"></i>
+                  </div>
+                  -->
+
+                  <div class="project-names col-xs-12 col-sm-5 pad-none">
```

Comment:
Quick comment here to doc our discussion, we should stick with either flex or bootstrap for the layout, but not mix.  So the above `<div row>` can be changed to `<div class="row">` if going with bootstrap children `<div class="col-md-5">`, or these child divs can be changed to something like `<div flex grow="2">` if you wanna keep the above `<div row>` to go all in with flex instead.